### PR TITLE
Removed toUpperCase function!

### PR DESCRIPTION
**Pull request by secrashi**
I kept getting an error that '_toUpperCase is not a function_' whenever I tried to load deprecated terms from the drop-down menu. 

WhatsApp Image 2021-04-24 at 6 34 54 PM

- I tried to look for possible solutions, and I got another error saying '_cannot read property 'toUpperCase of undefined_' , I realized that while loading deprecated terms, the value of a and b would possibly be an empty string. I tried to use the if-else situation, where it is not an empty string then `a=textAccessor(a).toUpperCase();`. But this too didn't work and I haven't yet figured out why. 
- Another possible solution that seemed to me was to remove toUpperCase function, though I wasn't sure, if it would affect other elements of the website. After implementing this, I checked if any other element was affected due to this, but I can't find any. 
Can @hmenager @bryan-brancotte @matuskalas let me know if this is the correct solution to the problem? If not, then can u give me a hint on which direction to look for?

### CODE REVIEW

**Comment by bryan-brancotte** [comment]
Hi @secrashi 

Thanks for spotting the issue, could you indicated what was your search of terms ?

The toUpperCase is need in order to sort while not beeing sensible to the case, without it names would be sorted like:
```
Alice
Charlie
Daniel
bob
eva
```

So here we have to option : 
 *  make sure `textAccessor(a)` returns something
 * use a default blank string when `textAccessor(a)` returns nothing

I would prefer to change the `textAccessor` implementation that sometime returns undefined values as it could also cause issue elsewhere later.

**Comment by secrashi** [comment]
Hello @bryan-brancotte, 
> Thanks for spotting the issue, could you indicated what was your search of terms ?
- I received this error `Uncaught TypeError: textAccessor(...).toUpperCase is not a function` when I tried to click the option '_Deprecated Terms_' and '_All branches and deprecated terms_' from the dropdown menu.


> So here we have to option :
> 
> * make sure `textAccessor(a)` returns something
> * use a default blank string when `textAccessor(a)` returns nothing
> 
> I would prefer to change the `textAccessor` implementation that sometime returns undefined values as it could also cause issue elsewhere later.

Sure, I'll look for ways to do this! Should I make a different PR for the changes or commit the changes here directly?

**Comment by bryan-brancotte** [comment]
> * I received this error `Uncaught TypeError: textAccessor(...).toUpperCase is not a function` when I tried to click the option '_Deprecated Terms_' and '_All branches and deprecated terms_' from the dropdown menu.

Thanks for the intel

File: js/tree-reusable-d3.js [hunk 1]
```diff
@@ -412,11 +412,13 @@ function interactive_tree() {
...
-                        b=textAccessor(b).toUpperCase();
-                        if (a < b)
+                        a=textAccessor(a).toString();
+                        a=a.toUpperCase();
+                        b=textAccessor(b).toString();
+                        b=b.toUpperCase();
+                        if (parseInt(a) < parseInt(b))
                             return -1;
-                        if (a > b)
+                        if (parseInt(a) > parseInt(b))
```
Comment (bryan-brancotte) [inline js/tree-reusable-d3.js:421]: I am not sure that is solves the issue as parseInt("ee") returns NaN.

**Review by bryan-brancotte — CHANGES_REQUESTED** [review]
Hi,
I don't think that `parseInt` is the solution. Before solving the issue, you should identify which node/element cause the problem. I let you search for it, ping me if you want some help

**Comment by bryan-brancotte** [comment]
work is being carried on in #181

**Pull request closed without merging**